src/_static/js/main.js

```diff
@@ -0,0 +1,64 @@
+window.addEventListener('WebComponentsReady', function documentReady() {
+  $('.button-collapse').sideNav();
+  $('.dropdown-button').dropdown();
+  $('ul.tabs').tabs();
+  $('.collapsible').collapsible();
+
+  $('.slider-carousel').owlCarousel({
```

Comment:
@andela-iakande We don't need most of these for the component.

Reply:
Ok. It has been removed